reproc/test/pid.c

```diff
@@ -0,0 +1,40 @@
+#include "assert.h"
+
+#include <reproc/drain.h>
+#include <reproc/reproc.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(void)
+{
+  const char *argv[] = { RESOURCE_DIRECTORY "/pid", NULL };
+  char *output = NULL;
+  reproc_sink sink = reproc_sink_string(&output);
+  int r = -1;
+
+  reproc_t *process = reproc_new();
+  ASSERT(process);
+
+  int child_pid = reproc_pid(process);
+  ASSERT(REPROC_EINVAL == child_pid);
```

Comment:
All conditionals in reproc have the constants on the right hand side so these two should switch sides

Reply:
Done.